pkg/common/providers/rosaprovider/cluster.go

```diff
@@ -144,6 +144,10 @@ func (m *ROSAProvider) LaunchCluster(clusterName string) (string, error) {
 		}
 	}
 
+	if viper.GetBool(config.Tests.EnableFips) {
```

Comment:
I know this config setting was already [present](https://github.com/openshift/osde2e/blob/main/pkg/common/config/config.go#L246) under test configuration. Do we think this should be a cluster specific setting under [cluster config](https://github.com/openshift/osde2e/blob/main/pkg/common/config/config.go#L265)?

Looks like this config setting is only used within the `pkg/e2e/verify/fips.go` test and there are 2 configs (fedramp/fips).

This change looks good to me :+1:, just wanted to mention this.

Reply:
updated in b558e8a

Reply:
Thanks for making the change @jbpratt!